### PR TITLE
Document `spacy-llm`'s `TranslationTask`

### PR DESCRIPTION
**Pull request by rmitsch**
## Description

Document `spacy-llm`'s `TranslationTask`. See https://github.com/explosion/spacy-llm/pull/396.

### Types of change

Doc update.

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [ ] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/docs/api/large-language-models.mdx [hunk 1]
```diff
@@ -236,6 +236,57 @@ objects. This depends on the return type of the [model](#models).
...
+> ```ini
+> [components.llm.task]
+> @llm_tasks = "spacy.Translation.v1"
+> examples = null
+> target_lang = "Spanish"
+> ```
+
+| Argument                    | Description                                                                                                                                                                                   |
+| --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `template`                  | Custom prompt template to send to LLM model. Defaults to [summarization.v1.jinja](https://github.com/explosion/spacy-llm/blob/main/spacy_llm/tasks/templates/summarization.v1.jinja). ~~str~~ |
```
Comment (svlandeg) [inline website/docs/api/large-language-models.mdx:259]: TODO: update reference to jinja template

File: website/docs/api/large-language-models.mdx [hunk 2]
```diff
@@ -236,6 +236,57 @@ objects. This depends on the return type of the [model](#models).
...
+| `examples`                  | Optional function that generates examples for few-shot learning. Defaults to `None`. ~~Optional[Callable[[], Iterable[Any]]]~~                                                                |
+| `parse_responses` (NEW)     | Callable for parsing LLM responses for this task. Defaults to the internal parsing method for this task. ~~Optional[TaskResponseParser[SummarizationTask]]~~                                  |
+| `prompt_example_type` (NEW) | Type to use for fewshot examples. Defaults to `SummarizationExample`. ~~Optional[Type[FewshotExample]]~~                                                                                      |
```
Comment (svlandeg) [inline website/docs/api/large-language-models.mdx:262]: Update c/p error referring to "summarization" twice more 😉

**Review by svlandeg — COMMENTED** [review]
Looks good, apart from some copy/paste errors.
Let's merge after the original `spacy-llm` PR is merged.

File: website/docs/api/large-language-models.mdx (hunk 2 above)
Comment (rmitsch) [reply website/docs/api/large-language-models.mdx]: :upside_down_face: Fixed in https://github.com/explosion/spaCy/pull/13183/commits/6265e0c52886525df21b39f8f762bfed91e7732d.

File: website/docs/api/large-language-models.mdx (hunk 1 above)
Comment (rmitsch) [reply website/docs/api/large-language-models.mdx]: Fixed in https://github.com/explosion/spaCy/commit/6265e0c52886525df21b39f8f762bfed91e7732d.

**Pull request merged**